organization/_posts/2020/2020-09-30-packaging.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "HSF Software Tools and Packaging Working Group Meeting #30, 30th September 2020"
+layout: meetings
```

Comment:
If you change to layout `plain` then the title is automatically included and doesn't need to be repeated below. (Yes, the names `plain` and `meetings` are really backwards here...)